pkg/compiler/codegen.go

```diff
@@ -915,6 +915,9 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 			}
 		case *ast.SelectorExpr:
 			name, isMethod := c.getFuncNameFromSelector(fun)
+			if isMethod && name[0] == '*' {
```

Comment:
It seems this branch is not covered.
Also, usage analysis should have similar checks, so can we move this `if` inside of `getFuncNameFromSelector`?

Reply:
Let me think about it, the branch itself should be covered by the `TestInlinedMethod` (it uses pointer receiver).

Reply:
However, we use the type of a local variable, not of a function formal receiver. I think using `z = &inline.T{}` should cover this.

Reply:
Yep, that works, added another test and reorder commits.